odoo/addons/test_read_group/tests/test_empty.py

```diff
@@ -61,3 +61,11 @@ def test_mixed(self):
             'date': 'December 1916',
             'value': 7,
         }])
+
+    def test_field_error(self):
+        Model = self.env['test_read_group.aggregate']
+        try:
+            Model.read_group([], ['value', 'not_another_field'], ['partner_id', 'not_a_field'])
+            raise AssertionError('Should have raised an error')
+        except ValueError as e:
+            self.assertEqual(str(e), "Invalid field 'not_another_field' on model 'test_read_group.aggregate'")
```

Comment:
You can use `with self.assertRaisesRegex(ValueError, "Invalid field 'not_another_field' on model 'test_read_group.aggregate'"):`.

Reply:
haa, that's what I wanted :D thanks :)